fun/lms/views.py

```diff
@@ -0,0 +1,24 @@
+import re
+from django.shortcuts import redirect
+from django.http import HttpResponse
+
+
+def richie(request):
+    RICHIE_ROUTE_REGEXP = r"^/richie/(?P<target>.*)"
+    # TODO use a settings to definie PLATFORM_RICHIE_URL
+    PLATFORM_RICHIE_URL = "https://richie.local.dev:8070"
+
+    def extract_richie_target(url):
```

Comment:
you don't need to do this, you can catch it in the url patterns route. For an example: https://docs.djangoproject.com/en/3.1/topics/http/urls/#example